follower_and_avoider:
  ros__parameters:

    ball_proportional_gain: 0.01
    ball_integral_gain: 0.0
    ball_derivative_gain: 0.0
    
    person_proportional_gain: 1.0
    person_integral_gain: 1.0
    person_derivative_gain: 1.0